kinder/ci/workflows/upgrade-1.12-1.13.yaml

```diff
@@ -0,0 +1,11 @@
+version: 1
+summary: |
+  this workflow test kubeadm upgrades from Kubernetes ci/latest-1.12 version to Kubernetes release/latest-1.13 version
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-all#kubeadm-kinder-upgrade-1-12-1-13
+  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml
+  config    > https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml
```

Comment:
while the summary/descriptions are nice to have, they require from us to update them when we are copying a workflow to create a new workflow from it.
how about using the vars in the summary?

```
this workflow test kubeadm upgrades from Kubernetes initVersion version to Kubernetes upgradeVersion version
  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-all#kubeadm-kinder-upgrade-initVersion-upgradeVersion
```

the same can apply to the other upgrade jobs.

Reply:
Creating a new test isn't a frequent operation, so IMO it is acceptable to change the description
(and, at least for me, changing a description helps in focus on what I'm doing)